src/login/logind-action.c

```diff
@@ -176,9 +176,9 @@ static int run_helper(const char *helper) {
         return wait_for_terminate_and_warn(helper, pid, true);
 }
 
-static int write_mode(char **modes) {
+static int write_mode(const char **modes) {
         int r = 0;
-        char **mode;
+        const char **mode;
 
```

Comment:
This will make the pointer to the char pointer const. What is this good for? The functions called with *mode take it as const char* anyway.